docs/resources/database_postgresql.md

```diff
@@ -0,0 +1,76 @@
+# Resource: <resource name> delphix_database_postgresql
+
+In Delphix, a databsae/source is required in the environment using which a dsource can be created.
+
+## System Requirements
+
+* Data Control Tower v14.0.1+ is required for dSource management. Lower versions are not supported.
```

Comment:
Data Control Tower v14.0.1+ is required for dSource management. Lower versions are not supported.

1) Do we expect there to be a patch? Or should this be v14.0.0?
2) This is Database (Source Config) management right? Not dSource?
3) Also, lower versions are just not supported by this Resource? But if we want to make that claim everywhere, then we need put something on the main Provider doc page

Reply:
This will not be a patch it will be 14.0.0 , yes this does not support lower versions we should put out on the main provider doc aswell